src/test/features/response/form/models/responseType.ts

```diff
@@ -0,0 +1,19 @@
+import { expect } from 'chai'
+import { ResponseType } from 'response/form/models/responseType'
+
+describe('ResponseType', () => {
+
+  describe('valueOf', () => {
+
+    it('should return ResponseType for corresponding value', () => {
+      expect(ResponseType.valueOf(ResponseType.OWE_ALL_PAID_NONE.value)).to.be.eq(ResponseType.OWE_ALL_PAID_NONE)
```

Comment:
Shall we make it more dynamic by iterating through all types?

Reply:
I don't mind. @rlewan, @timja-kainos what do you think? Do you like this idea?

Reply:
I'm already doing it for the residence question
```
    it('should return correct values', () => {
      ResidenceType.all().forEach((element) => {
        expect(ResidenceType.valueOf(element.value).value).to.equal(element.value)
      })
    })
```
so yes -- I like the idea ;)

Reply:
sure